### ISSUE TITLE
Precompilation error due to changes in Zygote.jl

### ISSUE THREAD

**Issue by JaydevSR** [JuliaMolSim/Molly.jl#79]
There seems to be some changes in Zygote.jl in the latest release in which the function `Zygote._backmean` was removed. Molly provides a redefinition of this function in https://github.com/JuliaMolSim/Molly.jl/blob/0846f9239a8b9a4613c6f900d8753b3c4d605c44/src/zygote.jl#L114

Due to this the package fails to precompile.

See the commit: https://github.com/FluxML/Zygote.jl/commit/af434d621e06a10b58aac5dc568312e4b144bb9a#diff-a9e025ac90a30d27e7512546971c5d92ea7c3496ba759336ae6bf1cace6db4b2

**Comment by jgreener64** [comment]
You are correct, I was waiting for the new Zygote to be tagged to fix this. I will look to fix it today, though we do have compat bounds on Zygote so it should be okay in the short term.
